woocommerce/woocommerce
# Add email type to email field in Checkout block

Previous to additional fields, the email field was hardcoded in the contact information step, this also included its type, when we abstracted that, we started using fields from CheckoutFields, however those didn't include a type. This PR includes that.

Closes https://github.com/woocommerce/woocommerce/issues/48610


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. From a mobile phone (or via the link to the playground), add a product to cart and go to Checkout.
2. Focus on Email field, make sure an email keyboard is shown (`@` is a top level key in the keyboard).
3. If you have access to an ipad, make sure it also shows provider autocomplete buttons.
4. You can also inspect the page and see if the field type is email and autocomplete is email.



### Changelog entry







#### Significance




#### Type




#### Message 

> Mark Checkout email field as type email to show relevant keyboard when filling.
